src/core/simulation_sequence.jl

```diff
@@ -132,50 +133,57 @@ end
 mutable struct SimulationSequence
     horizons::Dict{String, Int}
     step_resolution::Dates.TimePeriod
-    intervals::Dict{String, <:Dates.TimePeriod}
+    intervals::Dict{String, Tuple{<:Dates.TimePeriod, <:FeedForwardChronology}}
     order::Dict{Int, String}
     feedforward_chronologies::Dict{Pair{String, String}, <:FeedForwardChronology}
     feedforward::Dict{Tuple{String, Symbol, Symbol}, <:AbstractAffectFeedForward}
     ini_cond_chronology::IniCondChronology
     cache::Dict{String, Vector{<:AbstractCache}}
     execution_order::Vector{Int}
+    current_execution_index::Int64
 
     function SimulationSequence(;
         horizons::Dict{String, Int},
         step_resolution::Dates.TimePeriod,
-        intervals::Dict{String, <:Dates.TimePeriod},
+        intervals::Dict{String, <:Tuple{<:Dates.TimePeriod, <:FeedForwardChronology}},
```

Comment:
So, can the `intervalas[key][2]` be any `FeedForwardChrolongy`? I don't understand what the implications of a `Synchronize()` entry would be...

Reply:
Synchronize implies that the next execution of the same stage uses information from the last step in the interval to update its data.